target/linux/mvebu/base-files/etc/uci-defaults/04_mambafan

```diff
@@ -10,10 +10,20 @@ board=$(mvebu_board_name)
 case "$board" in
 armada-xp-linksys-mamba)
         # Set one execution of the fan script during every start (waiting for cron)
-        echo "/sbin/fan_ctrl.sh" > /etc/rc.local
+        if grep -q fan_ctrl.sh /etc/rc.local
+                then
+                        exit 0
+                else
+                        echo "/sbin/fan_ctrl.sh" > /etc/rc.local
```

Comment:
There is always of course the option of removing rc.local in total, but then user would have to wait until the first cron execution for the fan to spin down, which I don't like.